### ISSUE TITLE
안녕하세요~

### ISSUE THREAD

**Issue by hgh21** [seorim0/DCCRN-with-various-loss-functions#3]

**Comment by seorim0** [comment]
데이터는 제가 편의상 noisy, clean을 하나의 numpy 파일로 만들어서 썼어요!   
대략적으로 코드로 나타내면 아래와 같습니다.    
```
dataset = []
for i in range(data_num):  # data_num = the number of speech data
    dataset.append([noisy_data, clean_data])
```   

그리고 굳이 .npy를 만들 필요없이 wavfile을 바로 불러오셔도 되는데,   
그 경우에는 dataloader.py의 __getitem__(self, idx)를 아래와 같이 수정하여 쓰시면 됩니다.    

```
    def __getitem__(self, idx):
        # read the wav
        inputs = addr2wav(self.noisy_dirs[idx])
        targets = addr2wav(self.clean_dirs[idx])

        # transform to torch from numpy
        inputs = torch.from_numpy(inputs)
        targets = torch.from_numpy(targets)

        # (-1, 1)
        inputs = torch.clamp_(inputs, -1, 1)
        targets = torch.clamp_(targets, -1, 1)

        return inputs, targets
```   

메일로 질문 내용이 남아있어서 답변드려요 :)

**Comment by hgh21** [comment]
감사합니다~
